app/pods/publications/withdrawn/template.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="auk-scroll-wrapper__body">
+  <div class="auk-u-m-4">
+    <div class="auk-u-my-2">
+      <WebComponents::AuNavbar class="vlc-publication-overview-navbar" @noPadding="true">
```

Comment:
Gingen we niet afstappen van VLC css ?

Reply:
Ik heb de bestaande navbar gewoon overgenomen.
Lijkt me iets om apart op te pikken.
<img width="605" alt="Screenshot 2021-01-04 at 11 04 12" src="https://user-images.githubusercontent.com/592312/103523723-99e89580-4e7c-11eb-96f9-5920f47604ed.png">